### DISCUSSION TITLE
Testing Team Discussions

### DISCUSSION THREAD

**Discussion by ThomDietrich** [openhab/openhab-docs#2048] [Team Posts]
Hey guys.

https://github.com/blog/2471-introducing-team-discussions

**Comment by Confectrician** [comment]
Hi

**Comment by Confectrician** [comment]
Thats' pretty cool. So we are able to discuss topic related content in a separate discussion thread.

**Comment by ThomDietrich** [comment]
I like this very much! Would be fine by me to discuss wider topics here from now on.

**Comment by bgilmer77** [comment]
So gitter -1?

**Comment by Confectrician** [comment]
> So gitter -1?

I would suggest that.
I think staying in github directly is more comfortable and in a later point this might be useg openhab wide for all repos, so its fine for me to get used to it directly.

**Comment by ThomDietrich** [comment]
Yep!
